examples/identifier.kl

```diff
@@ -0,0 +1,9 @@
+#lang "prelude.kl"
+
+(define identifier?
+  (lambda (x)
+    (syntax-case x
+      [(ident x) (true)]
+      [_ (false)])))
```

Comment:
A few modules were defining this same helper function, so I moved it to its own module so I could use it in `fancy-syntax-case` as well.